src/modules/account/components/AccountDetails/AccountOverview/index.js

```diff
@@ -40,6 +40,7 @@ export default function AccountOverview() {
     active: activeTab,
     onClick: ({ value }) => setActiveTab(value),
   };
+  const userAddress = searchAddress ?? currentAddress;
```

Comment:
we never try to use term `user`, lets rename to `accountAddress` or just `address`?

Reply:
Fixed